io/net/multicast.py

```diff
@@ -44,19 +45,25 @@ def setUp(self):
         self.peer_password = self.params.get("peer_password",
                                              '*', default="None")
         interfaces = netifaces.interfaces()
-        self.iface = self.params.get("interface", default="")
+        self.hbond = self.params.get("hbond", default=False)
+        if not self.hbond:
+            self.iface = self.params.get("interface", default="")
+        else:
+            self.mac_id = self.params.get("mac_id", default="02:03:03:03:03:01")
+            self.iface = self.get_hbond(self.mac_id)
```

Comment:
You can avoid one else here
```
self.iface = self.params.get("interface", default="")
if self.hbond:
            self.mac_id = self.params.get("mac_id", default="02:03:03:03:03:01")
            self.iface = self.get_hbond(self.mac_id)
```

Reply:
@vaishnavibhat  main part,  Could you please check if the slave mac and bond mac are not the same in case of HNV bond.. otherwise it will clash.  also  you need to check the slave and bond mac address after migration too.

Reply:
> 1. as get_hbond gets called from all tests, why not move to net utils  ?
Plan is to move this function to utils in the future. 
> 2. Make sure the slaves do not have same mac
Can you please elaborate this one ? bond will take the mac of the active slave,, so there will be slave and bond with same mac,,

> 3. as the test are mac aligned, make sure the mac is not chaning after
>    a. Migration
>    b. DLPAR of any slave
>    c. failover
We have fixes for NetworkManager taking care of this one. 

> 4. FYI, The bond name, slave name, mac address will always change after migration.  ping check test will fail after migration.. lpm test should handle the new name change .. if we want to run all nic bucket after migration
are we starting the bucket after the migration or before ? If its after then the user should give the new mac addr in the yaml file,, if its started before, then t should not matter,,  we may have to try this during migration and check..